.meta/_partials/fields/_conditions_options.toml.erb

```diff
@@ -62,34 +66,40 @@ being `true` or `false` respectively.\
 [<%= namespace %>."`[field_name]`.contains"]
 type = "string"
 examples = [
-  { "message.contains" = "foo" }
+  { "message.contains" = "foo" },
+  { "message.contains" = ["foo","bar"] }
 ]
 common = true
 relevant_when = {type = "check_fields"}
 description = """\
 Checks whether a string field contains a string argument.\
+Can also provide a list of string (evaluates to true if any match).\
 """
 
 [<%= namespace %>."`[field_name]`.ends_with"]
 type = "string"
 examples = [
-  { "environment.ends_with" = "-staging" }
+  { "environment.ends_with" = "-staging" },
+  { "environment.ends_with" = ["-staging", "-running"] }
 ]
 common = true
 relevant_when = {type = "check_fields"}
 description = """\
 Checks whether a string field ends with a string argument.\
+Can also provide a list of string (evaluates to true if any match).\
 """
 
 [<%= namespace %>."`[field_name]`.starts_with"]
 type = "string"
 examples = [
-  { "environment.starts_with" = "staging-" }
+  { "environment.starts_with" = "staging-" },
+  { "environment.ends_with" = ["staging-", "running-"] }
```

Comment:
Is that last line a typo? The first one says `starts_with`, the second `ends_with`.

Reply:
Yeah typo -- whew good catch!